apps/communications/contacts/views/details/js/boot.js

```diff
@@ -45,6 +46,62 @@ window.onload = function() {
       document.body.classList.remove('hidden');
     });
 
+    function getContactInfo(id) {
+      return new Promise(function(resolve, reject) {
+        ContactsService.get(params.contact, function onSuccess(savedContact) {
+          ContactsService.getCount(function(count) {
+            resolve(
+              {
+                contact: savedContact,
+                count: count
+              }
+            );
+          });
+        }, function onError() {
+          console.error('Error retrieving contact');
+          reject();
+        });
+      });
+    }
+
+
+    // Get action from URL (new or update)
+    var params = ParamUtils.get();
+    if (params && params.contact) {
```

Comment:
I can not return here because after that we are adding the listener to activities.